### DISCUSSION TITLE
livebasah.eu.org - VPN Detected

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1483] [Q&A] [answered]
![20230731_231255_rmscr.jpg](https://github.com/realodix/AdBlockID/assets/114223791/65d01932-0eac-414d-806e-d26cd48db089)

Yang seperti ini maksudnya apa ya?
Ini bisa berpengaruh gk dengan adguard apk yg menggunakan filtering vpn lokal
Https filtering

**Comment by Jokopentil** [comment]
Haya menggunakan cloudflare

**Comment by realodix** [comment]
Linknya mana? Saya harus lihat langsung untuk menjawabnya

**Comment by Jokopentil** [comment]
https://livebasah.eu.org/pasangan-mesum-ngewe-di-alam-terbuka/?player=2

**Comment by realodix** [comment, marked as answer]
Playernya tidak mengizinkan penggunaan VPN.

> Haya menggunakan cloudflare

Mereka mengidentifikasi cloudflare warp sebagai VPN, jadi gak bisa.
